src/components/Section/SectionWrapper.js

```diff
@@ -53,9 +34,9 @@ const Wrapper = React.forwardRef(({ children, id }, ref) => {
   }
 
   return (
-    <SectionWrapper ref={ref && ref} id={id}>
+    <StyledSectionWrapper ref={ref && ref} id={id}>
```

Comment:
I've missed it some previously... but this `ref && ref` doesn't make sense. 
look: you have forward ref, so - type for ref will be `HTMLSomeSpecificElementInExampleDiv` or `null` (or `undefined`, but when we are using `useRef` we should pass `null` as default - Typescript could give you hint). 
So the result:
`undefined && undefined` -> `undefined`
`null && null` -> `null`
`element && element` -> `element`